scania-digital-design-system/sdds
# AB#380 -  Feature/textfield options

**Describe pull-request**  
_Describe what the pull-request is about_
Textfield component

- [x] Helper text
- [x] Error text
- [x] Error states
- [x] Disabled states
- [x] Label inside transition
- [x] Text counter
- [x] Suffix symbol
- [x] Prefix symbol
- [x] Icon Sufix
- [x] icon Prefix
- [x] Added comments for the web-component
- [x] Cleaned up the npm scripts a bit
- [x] Minor bug fixes
- [x] Focus on textfield when clicking the whole element

**Solving issue**  
_Add which issue this pull-request solves by adding # plus the number of the issue (for example #123)_
Fixes: #75 

**How to test**  
_Add description how to test if possible_
1. Run storybook or look at netlify
2. Check textfield options
3. Compare to figma examples
